src/scene/shader-lib/chunks/common/vert/transform.js

```diff
@@ -115,6 +115,9 @@ vec4 getPosition() {
     vec4 screenPos;
     #ifdef UV1LAYOUT
     screenPos = vec4(vertex_texCoord1.xy * 2.0 - 1.0, 0.5, 1);
+        #ifdef UV1LAYOUT_FLIP
+        screenPos.y *= -1.0;
+        #endif
```

Comment:
I'm curious, why this way while in other places we just use [`#ifdef WEBGPU`](https://github.com/search?q=repo%3Aplaycanvas%2Fengine+%22%23ifdef+WEBGPU%22&type=code) instead?

Reply:
good point, I'll change it to that too, thanks